### ISSUE TITLE
Feature request: redo.

### ISSUE THREAD

**Issue by Luaisa** [nvrmind/IssueTracker#46]

**Comment by daseyb** [comment]
You can actually redo, by clicking the right arrow in the brush menu (above the brush settings)

**Comment by StainyGrimes** [comment]
It could do with being more immediately accessible on the controller like undo is, but the trouble is where to put it.

**Comment by Luaisa** [comment]
Yes, in fact in the brush menu is not so user-friendly.  I'll let you know if I'll find some suggestions about this topic. Thanks for the prompt answer!
